Rocket.Chat/Views/Cells/Chat/ChatMessageTextViewModel.swift

```diff
@@ -21,12 +21,13 @@ protocol ChatMessageTextViewModelProtocol {
 }
 
 final class ChatMessageTextViewModel: ChatMessageTextViewModelProtocol {
-    var color: UIColor {
-        if let color = attachment.color {
+    var color: UIColor? {
+//        if let color = attachment.color {
```

Comment:
Can you remove the commented code please?

Reply:
@rafaelks done